sqlglot/generator.py

```diff
@@ -94,6 +94,9 @@ class Generator:
     ROOT_PROPERTIES = {
         exp.ReturnsProperty,
         exp.LanguageProperty,
+        exp.DistStyleProperty,
+        exp.DistKeyProperty,
+        exp.SortKeyProperty,
```

Comment:
Should we override `ROOT_PROPERTIES` for Redshift and add them there?

Reply:
yea let's add these to redshift